src/redisjson.rs

```diff
@@ -64,4 +64,30 @@ impl RedisJSON {
 
         return Ok(s)
     }
+
+    pub fn get_type(&self, path: &str) -> Result<String, Error> {
+        let s = match self.get_doc(path)? {
+            Some(doc) => {
+                match *doc {
```

Comment:
This is Rust... no need to dereference explicitly 😄 
Just `match doc {` will do nicely.